public/dist/overviewBundle.js.LICENSE.txt

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
once this gets in the repo, how can we get rid of it from master? I failed at gitignoring mine apparently